src/odemis/dataio/test/catmaid_test.py

```diff
@@ -56,7 +57,11 @@ def test_open_data_virtualflybrain(self):
         """
         # test for a url with a specified project id and stack id
         url = 'catmaids://fafb.catmaid.virtualflybrain.org/?pid=2&sid0=2'
-        acquisition = open_data(url)
+        try:
+            acquisition = open_data(url)
+        except ValueError:
+            logging.error("No stack info at url %s." % url)
+            self.skipTest("No stack info at url %s." % url)
```

Comment:
This looks like the URL is a dead link. Can you check with @tepals  for another server to use for testing?